### ISSUE TITLE
Update emailjs library files

### ISSUE THREAD

**Issue by sosnovsky** [FlowCrypt/flowcrypt-browser#5344]
Yeah, but I've made various fixes in the library without upstreaming them. So it may be a messy process. Hopefully we have tests for that. But right now, I recommend to just bring the patch here and file updating the library for another issue.

_Originally posted by @tomholub in https://github.com/FlowCrypt/flowcrypt-browser/issues/5310#issuecomment-1662204817_

**Comment by sosnovsky** [comment]
`emailjs` library was deprecated and replaced by https://github.com/nodemailer/libmime/, so we should use it instead as it's already includes fixes for some issues which we fixed manually (like encoding filenames with utf-8 symbols and emojis)

**Comment by tomholub** [comment]
yeah but that sounds like quite the task. I'd wait at least until manifest v3 is implemented.

**Comment by tomholub** [comment]
and i'd wait me even after that, unless there is particular pain point with email js

**Comment by sosnovsky** [comment]
I was trying to fix encoding of attachment names with special symbols, but manually fixing `emailjs` code still caused other bugs. I checked https://github.com/nodemailer/libmime/ code and it's quite similar to `emailjs`, so I replaced functions related to attachments encoding with similar ones from `libmime` and it works well.

I tested encrypted messages and password-protected messages on FES - attachment names encoded well in both cases.
Maybe for now it'll be enough, but if there will be any issues with mime encoding/decoding - we can just check updated code from `libmime` and test if it fixes found issue.

**Comment by tomholub** [comment]
i see. ok
